.github/workflows/linux.yml

```diff
@@ -424,6 +429,12 @@ jobs:
             python3 -m pip install $wheel_name
           popd
 
+      - name: Install OpenVINO Python wheel from pre-built artifacts
```

Comment:
Can we change the wheel path only and do not introduce the new conditional step?

Reply:
We can, but I separated it to two steps 'cause it seemed a bit more readable to me though